bot/cogs/alias.py

```diff
@@ -74,20 +74,18 @@ def __init__(self, bot):
             self, ctx: Context, user: User, *, reason: str
     ):
         """
-        Alias for invoking <prefix>bigbrother watch user [text_channel].
+        Alias for invoking <prefix>bigbrother watch user reason.
```

Comment:
should maybe `reason` here be encased in some sort of brackets to indicate it's not supposed to be written literally? kind of like how text_channel was in square brackets?

Reply:
Done.